deploy/chaos-scheduler.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       labels:
         name: chaos-scheduler
     spec:
-      serviceAccountName: litmus
+      serviceAccountName: scheduler
```

Comment:
While installing chaos-operator the `litmus` service account will be created we can use that instead creating new, Is there  any permission difference?